.upstream-tests/utils/libcudacxx/compiler.py

```diff
@@ -146,6 +146,9 @@ def _initTypeAndVersion(self):
         if self.type == 'nvcc':
             # Treat C++ as CUDA when the compiler is NVCC.
             self.source_lang = 'cu'
+        elif self.type == 'clang':
+            # Treate C++ as clang-cuda when the compiler is Clang.
```

Comment:
typo ("treate")

Also "treat c++ as clang-cuda" doesn't particularly make sense, did you mean to say "as CUDA" as above?